pkgs/applications/science/math/R/default.nix

```diff
@@ -89,6 +91,13 @@ stdenv.mkDerivation (finalAttrs: {
 
   installTargets = [ "install" "install-info" "install-pdf" ];
 
+  # move tex files to $tex for use with texlive.combine
+  # add link in $out since ${R_SHARE_DIR}/texmf is hardcoded in several places
+  postInstall = ''
+    mv -T "$out/lib/R/share/texmf" "$tex"
```

Comment:
Would you make a subdirectory like `$tex/R` and put things into it?

Reply:
R already uses the "TeX Directory Structure", so the files are already in the correct location (which is `tex/latex` for the packages, `bibtex/...` for the bibtex stuff).

Reply:
OK. Thanks for clarification.